config/locales/wagon.de.yml

```diff
@@ -1678,6 +1678,8 @@ de:
     global:
       sac_cas/role/mitglied_common:
         no_permission: Für einen Austritt musst du dich an den Mitgliederdienst der Sektion wenden
+      group/abo_magazin/abonnent:
+        no_permission: "Meldung: “Eine Kündigung kann nur über den Mitgliederdienst des SAC erfolgen. Bitte schreibe eine E-Mail mit dem gewünschten Kündigungstermin sowie deinem Vor- und Nachnamen an mv@sac-cas.ch.”"
```

Comment:
Soll es wirklich Meldung: "..." sein? wirkt sehr komisch auf mich. Vor allem da es oben nicht so gemacht ist "Für einen Austritt musst du dich an den Mitgliederdienst der Sektion wenden"

Reply:
Habe ich mich auch gefragt, habe es vorerst 1 zu 1 aus dem Ticket übernommen, das sollte man eventuell nochmal abklären :+1:

Reply:
Habe es abgeklärt, es soll effektiv nur der Text in den Anführungszeichen angezeigt werden